docs/concepts/_index.md

```diff
@@ -49,10 +46,10 @@ learn more about all of these concepts in the [composition documentation].
 
 <!-- Named Links -->
 
-[Packages]: packages.md
+[Packages]: {{<ref "packages" >}}
```

Comment:
Is the `ref` syntax required? Not a _huge_ deal but is kind of a bummer that they don't render as markdown links on GitHub.

Reply:
No, it's not, but to the earlier comment, it's an easy and fast way to detect broken links. 

Although it doesn't render on github I think you'll get more value by running hugo locally and using that as a method for review, since that provides the visuals as well.